octobot_trading/personal_data/positions/position.py

```diff
@@ -305,7 +311,12 @@ def update_from_order(self, order):
 
         # Updates position average entry price from order only when increasing position side
         if self._is_update_increasing_size(size_update):
+            if self.size == constants.ZERO:
+                self.first_entry_time = self.exchange_manager.exchange.get_exchange_current_time()
```

Comment:
```suggestion
                self.creation_time = self.exchange_manager.exchange.get_exchange_current_time()
```
(linked to a previous comment)